Better behavior for the Shift functionality

eordano opened the pull request.

eordano committed:
Better behavior for the Shift functionality

nicosantangelo commented:
The modal while holding shift works great, but for what I can tell this is missing the rest of the features the implementation requires.

For example:
- should display after first bid
Explanied here:
![image](https://user-images.githubusercontent.com/692077/33882515-a96a8ede-df38-11e7-9daf-6caf55f1663a.png)

- should display when shift is pressed only
- should appear explained on the intro modal

eordano commented:
I'll take a swing at this later

eordano committed:
Only show notification after first bids

eordano commented:
Updated.

* Left out the explanation on the intro due to lack of copy
* If user has a pending bid, shift should show the notification already (is this ok?)
* Added a listener for focus of window to prevent weird issue with key still pressed
